### DISCUSSION TITLE
[REQUEST] Bloqueo de cuenta por intentos fallidos.

### DISCUSSION THREAD

**Discussion by manucabral** [TeamMrWeb/parches-chat#34] [Ideas]
Al intentar ingresar su cuenta y fallar datos de ingreso varias veces que la cuenta se bloquee y manda un alerta al mail o teléfono.

**Comment by manucabral** [comment]
La idea seria dar tantos intentos a cada cliente e identificarlo con una id, al momento que pase esos intentos será bloqueado en toda petición a la API

**Comment by larayavrs** [comment]
Entonces este bloqueo vendría siendo general? Lets go then.

**Comment by manucabral** [comment]
La idea lo propuso @AlexisViera lo transferí a un issue, por lo que entiendo seria un bloqueo total de la cuenta hasta que se pueda recuperar mediante e-mail o si llegamos a tener un buen sistema de mensajería por número de celular.
